pkg/ctl/create/utils.go

```diff
@@ -50,3 +51,44 @@ func checkVersion(ctl *eks.ClusterProvider, meta *api.ClusterMeta) error {
 
 	return nil
 }
+
+// loadSSHKey loads the ssh public key specified in the NodeGroup. The key should be specified
+// in only one way: by name (for a key existing in EC2), by path (for a key in a local file)
+// or by its contents (in the config-file). It also assumes that if ssh is enabled (SSH.Allow
+// == true) then one key was specified
+func loadSSHKey(ng *api.NodeGroup, clusterName string, provider api.ClusterProvider) error {
```

Comment:
Why not make it part of `pkg/ssh`?

Reply:
I put it here because I consider this to be on a different layer. This functionality does logic related to how the CLI and the config file should work (closer to the UI), while the other package should focus, IMO, on how to do SSH related things, without caring about the fine details of the user interface.

Reply:
👍